### DISCUSSION TITLE
When dpgen ran the first loop 02.fp, there was an error as follows. I can't solve it. Please help me, thank you

### DISCUSSION THREAD

**Discussion by 12jscvb** [deepmodeling/dpgen#1379] [Q&A]
The following is the screenshot of my error, thank you for your help
![2023-10-29](https://github.com/deepmodeling/dpgen/assets/145745956/07185274-1c40-4408-a18d-63b84132ad96)

**Comment by 12jscvb** [comment]
I found in the first cycle of 02 fp folder to appear error in the output file, its content as' 22 a79916470d3d3d165c49e7c49a7f2a9ce30005. Sub: line: 30 mpirun: command not found', but I don't know what that means. the following is screenshot
![2023-10-29 (1)](https://github.com/deepmodeling/dpgen/assets/145745956/0019fc1a-0c71-464e-9aa5-5a5ed540f85c)

**Comment by robinzyb** [comment]
this means there is no parallel computation command `mpirun` in your machine or where the calculation of cp2k should be carried out.

**Comment by 12jscvb** [comment]
> this means there is no parallel computation command `mpirun` in your machine or where the calculation of cp2k should be carried out.

I input the fp command in the machine.json file as follows, but it still reports an error. May I ask if the reason is that the cp2k environment is not properly configured. thank you for help
![2023-10-30 (3)](https://github.com/deepmodeling/dpgen/assets/145745956/ffe4e195-6234-4ed5-a36b-975b83236363)

**Comment by robinzyb** [comment]
have you checked whether the command can be run properly manually?

**Comment by 12jscvb** [comment]
> have you checked whether the command can be run properly manually
I found correctly entering the command below, but it still reports the same error
![2023-10-30 (8)](https://github.com/deepmodeling/dpgen/assets/145745956/fc9d6b09-298c-4706-a1d1-aae69e0fe6c8)

**Reply by robinzyb** [reply]
please manually submit a single cp2k job with this command. It looks to me that no mpirun is installed, which is not a problem of dpgen.
